openfisca_web_api/handlers.py

```diff
@@ -36,6 +38,32 @@ def calculate(tax_benefit_system, input_data):
     return input_data
 
 
+def dependencies(tax_benefit_system, input_data):
+    SimulationBuilder().build_from_entities(tax_benefit_system, input_data)
```

Comment:
Unless we use this line to get a simulation and run the payload? Running the simulation will get us the exact calculation period needed for each variable from the simulation holders.

For example, if we call a formula like the [housing_tax](https://demo.openfisca.org/legislation/housing_tax) formula, the called variable isn't at the simulation period but at some offset from the simulation period (`January` of the same year for the `accommodation_size` called by the `housing_tax` formula). This information isn't given to `/dependencies` users yet according to the current output format.
> Output example: `{"birth":3,"salary":2,"accommodation_size":1,"housing_occupancy_status":1}`

So, if we define an `accommodation_size` for another period, it will not be taken into account in the formula calculation and we won't get the expected results. 
To have this information, we would need to change the JSON response format to add the period to every variable. 

What do you think? Will you need to run a simulation out of the list of dependencies? 
If yes, this might require some information on how to set the variables values/some information on the period.